src/services/idGenerator.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const idGenerator = (arr) => {
+  if (!arr.length) {
+    return 1;
+  }
+
+  const maxId = Math.max(...arr.map(({ id }) => id));
+
+  return maxId + 1;
+};
```

Comment:
You can use [uuid](https://www.npmjs.com/package/uuid)

Reply:
Tests want id to be number